index.bs

```diff
@@ -1877,6 +1877,40 @@ If the bytes in the buffer have a natural intepretation
 as one of the other TypedArray types, provide that instead.
 For example, if the bytes represent Float32 values, use a {{Float32Array}}.
 
+<h3 id="allow-microtask-switches">A Promise represents a value, not a callback</h3>
+
+Avoid requiring that something needs to happen synchronously within
+a promise resolution or rejection callback.
+
+Authors tend to expect that any `async` function can be wrapped
+in another that appends synchronous code at the end e.g. to inspect state
+using try/catch. But the `await` queues a microtask:
```

Comment:
```suggestion
in another that appends synchronous code at the end.
This might be so that a return value can be examined, such as using try/catch.
Using `await` or similar wrappers queues a microtask:
```